site/docs/aws.md

```diff
@@ -238,6 +242,59 @@ LOCATION 's3://my-special-table-bucket'
 PARTITIONED BY (category);
 ```
 
+### DynamoDB Catalog
+
+Iceberg supports using a [DynamoDB](https://aws.amazon.com/dynamodb) table to record and manage database and table information.
+
+#### Configurations
+
+The DynamoDB catalog supports the following configurations:
+
+| Property                          | Default                                            | Description                                            |
+| --------------------------------- | -------------------------------------------------- | ------------------------------------------------------ |
+| dynamodb.table-name               | iceberg                                            | name of the DynamoDB table used by DynamoDbCatalog     |
+
+
+#### Internal Table Design
+
+The DynamoDB table is designed with the following columns:
+
+| Column            | Key             | Type        | Description                                                          |
+| ----------------- | --------------- | ----------- |--------------------------------------------------------------------- |
+| identifier        | partition key   | string      | table identifier such as `db1.table1`, or `NAMESPACE` for namespaces |
```

Comment:
sorry what do you mean by "or `NAMESPACE` for namespaces"? this `identifier` can be just the namespaces for sharing a single table with all tables within namespaces? And for the namespace case, is that always be this exact "NAMESPACE" string, or this is just an example?

Reply:
yes, that exact string